src/components/TodoHeader.vue

```diff
@@ -0,0 +1,21 @@
+<template>
+  <div>
+    <input v-model="newTodoName"/>
+    <button @click="$emit('addTodo', newTodoName)">
+      +
+    </button>
+  </div>
+</template>
+
+<script>
+import Vue from 'vue';
+
+export default Vue.component('TodoHeader', {
+  props: ['newTodoName'],
```

Comment:
props and data have the same variable `newTodoName` we can't have them together in several places. Please use in props or in data